app/shared/reservation-controls/ReservationControls.js

```diff
@@ -112,6 +113,16 @@ class ReservationControls extends Component {
         return [buttons.cancel];
       }
 
+      case 'waiting_for_payment': {
+        if (isAdmin) {
+          return isStaff
+            ? [buttons.cancel, buttons.edit]
+            : [buttons.cancel];
+        }
+        return [buttons.cancel, buttons.pay];
+      }
+
+      // does this state exist?
```

Comment:
It doesn't, so we might as well remove it here. Doesn't have to be changed in this PR, though.